shared/chat/conversation/container.tsx

```diff
@@ -53,7 +53,7 @@ let Conversation = (p: SwitchProps) => {
     // eslint-disable-next-line
     React.useEffect(() => {
       if (isFocused) {
-        if (_storeConvoIDKey !== conversationIDKey) {
+        if (_storeConvoIDKey !== conversationIDKey && Constants.isValidConversationIDKey(conversationIDKey)) {
```

Comment:
this was a race that changed. this is the minimal fix. if you have a pending thread we don't want to try and select it cause the valid thread can be coming in and replacing it